### PR TITLE
Update piped-base to 0.1.4

### PR DESCRIPTION
**Pull request by nakabonne**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: WORKSPACE [hunk 1]
```diff
@@ -133,7 +133,7 @@ container_pull(
     digest = "sha256:e863054859305080e89741bad1444e46a05ecb57cf2faf33a4bd448a992c47f6",
```
Comment (nghialv) [inline WORKSPACE:133]: SHA too.

Comment (nakabonne) [reply WORKSPACE:133]: I got wrong again.

**Comment by nakabonne** [comment]
Fixed

**Pull request merged**